### PR TITLE
Create a mechanism to read the released classes #17

### PR DESCRIPTION
**Pull request by soimugeoWB**
first need to merge backend-api MR https://github.com/protegeproject/webprotege-backend-api/pull/11

### CODE REVIEW

File: src/main/java/edu/stanford/protege/webprotege/project/IcdReleasedEntityStatusManagerImpl.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    private final OkHttpClient httpClient = new OkHttpClient();
+
+    private final Logger logger = LoggerFactory.getLogger(IcdReleasedEntityStatusManagerImpl.class);
+
+
+    public IcdReleasedEntityStatusManagerImpl(){}
+
+    @Override
+    public List<String> fetchIris() {
```
Comment (alexsilaghi) [inline src/main/java/edu/stanford/protege/webprotege/project/IcdReleasedEntityStatusManagerImpl.java:31]: [Question] will these ever change ? if yes we need to think about a way to do cache bust

**Review by alexsilaghi — APPROVED** [review]
it seems that the check pipeline was unable to compile the code.

File: src/main/java/edu/stanford/protege/webprotege/project/IcdReleasedEntityStatusManagerImpl.java (hunk 1 above)
Comment (soimugeoWB) [reply src/main/java/edu/stanford/protege/webprotege/project/IcdReleasedEntityStatusManagerImpl.java:31]: yes, they will change from time to time. at most 4 times a year from what i understood.

**Comment by soimugeoWB** [comment]
Yup, i think the checks failed because it cannot find the changes in backend-api. https://github.com/protegeproject/webprotege-backend-api/pull/11 once we merge the backend-api MR i'll update the version here.

**Pull request merged**